src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DynamicDocumentContainer.cs

```diff
@@ -9,7 +9,7 @@ internal abstract class DynamicDocumentContainer
     {
         public abstract string FilePath { get; }
 
-        public abstract bool SupportsDiagnostics { get; set; }
+        public virtual bool SupportsDiagnostics { get; set; }
```

Comment:
Why is abstract not ok?

Reply:
This was a little while back, but I believe it was so we could override the getter without having to `no-op` the setter.
https://github.com/dotnet/aspnetcore-tooling/pull/2501#discussion_r488973199

Reply:
Hmm, feels odd but if you want to go with it we can try it out